dist/scss/wds-components/_tabs.scss

```diff
@@ -45,7 +47,9 @@
 		transition: border-color .1s;
 
 		.wds-tabs__tab-label {
-			padding-bottom: $tab-padding-vertical - 2px;
+			// 4px is a height needed to align text after adding 2px bottom-border - it is 2 * 2px
+			// (we need to subtract 2px from bottom and top of the label container for text to be aligned properly)
+			height: $tab-height - 4px;
```

Comment:
can we always show border with transparent color? then we would not have to calculate the size here